### ISSUE TITLE
Conflicting argparse arguments

### ISSUE THREAD

**Issue by naveedhd** [stonier/rqt_py_trees#8]
```

Traceback (most recent call last):
  File "/opt/groot/desktop_ws/devel/bin/rqt_py_trees", line 6, in <module>
    exec(fh.read())
  File "<string>", line 22, in <module>
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rqt_gui/main.py", line 59, in main
    return super(Main, self).main(argv, standalone=standalone, plugin_argument_provider=plugin_argument_provider, plugin_manager_settings_prefix=str(hash(os.environ['ROS_PACKAGE_PATH'])))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/qt_gui/main.py", line 197, in main
    self.add_arguments(parser, standalone=bool(standalone), plugin_argument_provider=plugin_argument_provider)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/qt_gui/main.py", line 147, in add_arguments
    plugin_argument_provider(parser)
  File "/opt/groot/desktop_ws/src/rqt_py_trees/src/rqt_py_trees/behaviour_tree.py", line 307, in add_arguments
    operate_object.add_argument('-d', '--bag-dir', action='store', help='Specify the directory in which to look for bag files. The default is $ROS_HOME/behaviour_trees, if $ROS_HOME is set, or ~/.ros/behaviour_trees otherwise.')
  File "/usr/lib/python2.7/argparse.py", line 1297, in add_argument
    return self._add_action(action)
  File "/usr/lib/python2.7/argparse.py", line 1498, in _add_action
    action = super(_ArgumentGroup, self)._add_action(action)
  File "/usr/lib/python2.7/argparse.py", line 1311, in _add_action
    self._check_conflict(action)
  File "/usr/lib/python2.7/argparse.py", line 1449, in _check_conflict
    conflict_handler(action, confl_optionals)
  File "/usr/lib/python2.7/argparse.py", line 1456, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument -d/--disable-init-threads: conflicting option string(s): -d
```

**Comment by naveedhd** [comment]
This is caused due to update in `qt_gui_core` [here](https://github.com/ros-visualization/qt_gui_core/pull/69/files)

**Comment by stonier** [comment]
Resolved by #9.
